src/app/account/settings/tabs/password-and-authentication-tab/password-and-authentication-tab.component.spec.ts

```diff
@@ -23,9 +23,14 @@ describe("PasswordAndAuthenticationTabComponent", () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            declarations: [PasswordAndAuthenticationTabComponent, AdminChangePasswordComponent],
             providers: [Apollo],
-            imports: [HttpClientTestingModule, ToastrModule.forRoot(), MatDividerModule],
+            imports: [
+                HttpClientTestingModule,
+                ToastrModule.forRoot(),
```

Comment:
Toastr recommends switching to standalone providers:

![image](https://github.com/user-attachments/assets/53e65e0b-fa20-494a-8e6b-36883811556d)

Reply:
Changed